app/_includes/head.html

```diff
@@ -49,7 +49,6 @@
   <link rel="dns-prefetch" href="//cdn.segment.com">
   <link rel="dns-prefetch" href="//sidecar.gitter.im">
   <link rel="dns-prefetch" href="//buttons.github.io">
-  ##preload_assets##
```

Comment:
@thefosk, I pulled your branch and restored the token - I can see one situation where you may experience this: if you save a page, then `gulp jekyll` runs and browser-sync refreshes the page before `gulp html` finishes:

```bash
[21:23:32] Finished 'jekyll' after 3.93 s
[21:23:32] Starting 'html'...
[ ▶ ] Reloading Browsers...              <--- page refresh
[21:23:43] all files 11.92 MB
[21:23:43] Finished 'html' after 11 s .  <-- ##preload_assets## gets replaced, no auto refresh
```
If you manually refresh after `html` finishes, the token should disappear. Please try again and let me know if that works.